Example/ParalayoutSnapshotTests/ViewDistributionSnapshotTests.swift

```diff
@@ -268,6 +268,63 @@ final class ViewDistributionSnapshotTests: SnapshotTestCase {
         assertSnapshot(of: containerView, as: .image, named: nameForSnapshot(with: ["vertical"]))
     }
 
+    @MainActor
+    func testFixedDistributionProxy() {
+        let containerView = UIView(frame: CGRect(x: 0, y: 0, width: 400, height: 200))
+        containerView.backgroundColor = .white
+
+        let firstView = UIView(frame: CGRect(x: 0, y: 0, width: 50, height: 50))
+        firstView.backgroundColor = .blue.withAlphaComponent(0.5)
+        containerView.addSubview(firstView)
+
+        let secondView = UIView(frame: CGRect(x: 0, y: 0, width: 50, height: 50))
+        secondView.backgroundColor = .blue.withAlphaComponent(0.5)
+        containerView.addSubview(secondView)
+
+        let firstProxy = FixedDistributionProxy(length: 24)
+        let firstProxyView = UIView(frame: .zero)
+        firstProxyView.backgroundColor = .red.withAlphaComponent(0.5)
+        containerView.addSubview(firstProxyView)
+
+        let secondProxy = FixedDistributionProxy(length: 48)
+        let secondProxyView = UIView(frame: .zero)
+        secondProxyView.backgroundColor = .green.withAlphaComponent(0.5)
+        containerView.addSubview(secondProxyView)
+
+        containerView.semanticContentAttribute = .forceLeftToRight
+        containerView.applyHorizontalSubviewDistribution(inRect: containerView.bounds.insetBy(left: 10, top: 20, right: 30, bottom: 40)) {
+            firstView
+            firstProxy
+            secondView
+            secondProxy
+        }
+        firstProxyView.frame = firstProxy.rect
+        secondProxyView.frame = secondProxy.rect
+        assertSnapshot(of: containerView, as: .image, named: nameForSnapshot(with: ["horizontal", "LTR"]))
+
+        containerView.semanticContentAttribute = .forceRightToLeft
+        containerView.applyHorizontalSubviewDistribution(inRect: containerView.bounds.insetBy(left: 10, top: 20, right: 30, bottom: 40)) {
+            firstView
+            firstProxy
+            secondView
+            secondProxy
+        }
+        firstProxyView.frame = firstProxy.rect
+        secondProxyView.frame = secondProxy.rect
```

Comment:
in practice, this doesn't feel much different from putting `firstProxyView` and `secondProxyView` in the distribution above after giving them a fixed width.

In the end of the day, it's about the same number of lines to use a proxy vs not use a proxy. What exactly are we making simpler with this API?

Reply:
Oh, 100%. This is an incredibly trivial example for the sake of testing, and definitely would not be worth using a proxy for.